nextcloudappstore/api/v1/views.py

```diff
@@ -44,6 +44,11 @@ class NextcloudReleaseView(ListAPIView):
     serializer_class = NextcloudReleaseSerializer
 
 
+class AppsView(ListAPIView):
+    queryset = App.objects.all()
```

Comment:
You want to add prefetching, otherwise this will create thousands of SQL queries (n+1 problem), see https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/api/v1/views.py#L56 for a list of required relations that should be prefetched. See https://docs.djangoproject.com/en/2.1/ref/models/querysets/#prefetch-related and possibly https://docs.djangoproject.com/en/2.1/ref/models/querysets/#select-related

Reply:
You could extract that list into a global/singleton btw and just reuse it in both places.

Reply:
Did it like that - could you check again?